Resources/Textures/SimpleStation14/Clothing/Head/Hardsuits/hopsuit.rsi/meta.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": 1,
+  "license": "",
+  "copyright": "",
```

Comment:
These need to include your license, and copyright information. Since these are new sprites by you, license can be anything (Just use one of the default 14 ones, you'll find it in most files), and the copyright should just credit you.